src/ConfigEditor/ConfigEditor.tsx

```diff
@@ -209,6 +211,21 @@ export function ConfigEditor(props: Props) {
         label={selectors.components.ConfigEditor.Database.input}
         data-testid={selectors.components.ConfigEditor.Database.testID}
       />
+      <InlineField label="Send events to Amazon EventBridge" labelWidth={28}>
+        <Switch
+          value={props.options.jsonData.withEvent ?? false}
+          onChange={(e) =>
+            props.onOptionsChange({
```

Comment:
can you add a simple test for this? If you go to ConfigEditor.test.tsx you can take inspiration from the test `should allow user to enter a database`. You will need to add a `data-testid` to this Switch to test it.

Reply:
@andresmgot I have added test for this, If you see the test currently fails because the `onChange` is not getting triggered. I am unable to find a reason for this. Need your help in getting this test case passing. Thanks in advance! :-)

Reply:
I think it's because how `Switch` is internally implemented, if you do `fireEvent.click(withEventField);` rather than `fireEvent.change(withEventField, { target: { checked: true } });` it should work.